roles/linux/opensearch/templates/tlsconfig.yml

```diff
@@ -2,13 +2,13 @@ ca:
    root:
       dn: CN=root.ca.{{ domain_name }},OU=CA,O={{ domain_name }}\, Inc.,DC={{ domain_name }}
       keysize: 2048
-      validityDays: 730
+      validityDays: {{ cert_valid_days }}
       pkPassword: none
       file: root-ca.pem
 
 ### Default values and global settings
 defaults:
-      validityDays: 730
+      validityDays: {{ cert_valid_days }}
```

Comment:
Same as this existing PR @ssi444 
https://github.com/opensearch-project/ansible-playbook/pull/63/files
Should we revisit this until the other is merged ?
@saravanan30erd @peterzhuamazon

Reply:
The code is divided into logical blocks at @saravanan30erd request, so there are 3 PR's. Practically, you can fill in the last one. @prudhvigodithi